### PR TITLE
Mule Agent Plugin v2.1.8 release notes

### PR DESCRIPTION
**Pull request by Zamus**

### CODE REVIEW

File: modules/ROOT/pages/runtime-manager-agent/runtime-manager-agent-2.1.8-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+== Changes in This Release
+
+This release contains no changes to existing functionality.
+
+== Fixed Issues
+
+This release contains the following issue fixes:
+
+* Connection issues when placing a proxy in between the agent and the platform.
```
Comment (mcocaro) [inline modules/ROOT/pages/runtime-manager-agent/runtime-manager-agent-2.1.8-release-notes.adoc]: * Fixed connectivity issue with Anypoint Platform Control Plane when running the Agent behind a proxy

File: modules/ROOT/pages/runtime-manager-agent/runtime-manager-agent-2.1.8-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+_November 12, 2018_
+
+This document describes new features and enhancements, known limitations, issues, and fixes in Anypoint Runtime Manager Agent, version 2.1.8.
+
+== New Features and Enhancements
+
+This release contains the following features or enhancements:
+
+* Improved memory handling so that agent takes less time to startup.
```
Comment (mcocaro) [inline modules/ROOT/pages/runtime-manager-agent/runtime-manager-agent-2.1.8-release-notes.adoc]: * Faster startup time with less memory usage than earlier versions

File: modules/ROOT/pages/runtime-manager-agent/runtime-manager-agent-2.1.8-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+== Fixed Issues
+
+This release contains the following issue fixes:
+
+* Connection issues when placing a proxy in between the agent and the platform.
+
+== Known Limitations and Workarounds
+
+There are no known limitations or workarounds in this release.
```
Comment (mcocaro) [inline modules/ROOT/pages/runtime-manager-agent/runtime-manager-agent-2.1.8-release-notes.adoc]: *warn* about non backwards compatibility of this agent version

Comment (Zamus) [reply modules/ROOT/pages/runtime-manager-agent/runtime-manager-agent-2.1.8-release-notes.adoc]: @fpiagent Just to re-confirm here, this agent version DOES NOT have backward compatibility issues, am I right?

Comment (fpiagent) [reply modules/ROOT/pages/runtime-manager-agent/runtime-manager-agent-2.1.8-release-notes.adoc]: Confirmed. This agent works with both 4.2.X and 4.1.X Runtimes. We ran thorough tests to validate backwards compatibility and many bugs were fixed to achieve this.

**Pull request merged**